src/ConvertToSarif/ConvertToSarif.csproj

```diff
@@ -7,6 +7,15 @@
     <AssemblyName>ConvertToSarif</AssemblyName>
     <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <Prefer32Bit>true</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
+    <Prefer32Bit>true</Prefer32Bit>
+    <PlatformTarget>x86</PlatformTarget>
```

Comment:
Did your change to native.props not fix that? I thought that's what that change was for.

---
In reply to: [134336111](https://github.com/Microsoft/sarif-sdk/pull/590#discussion_r134336111) [](ancestors = 134336111)